modules/installation-user-defined-tags-azure.adoc

```diff
@@ -3,78 +3,16 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="installing-azure-user-defined-tags_{context}"]
-= Configuring user-defined tags for Azure
+= Configuring user-defined tags for {azure-short}
 
-In {product-title}, you can use the tags for grouping resources and for managing resource access and cost. You can define the tags on the Azure resources in the `install-config.yaml` file only during {product-title} cluster creation. You cannot modify the user-defined tags after cluster creation.
+In {product-title}, you can use tags for grouping resources and for managing resource access and cost. Tags are applied only to the resources created by the {product-title} installation program and its core operators such as Machine API Operator, Cluster Ingress Operator, Cluster Image Registry Operator. The {product-title} consists of the following types of tags:
```

Comment:
I am not sure what this means:

`The {product-title} consists of the following types of tags:`

> The OpenShift Container Platform consists of the following types of tags:

"OpenShift Container Platform uses the following types of tags" maybe?

Reply:
It means "OCP can have the following types of tags".